spec/models/claim_spec.rb

```diff
@@ -621,4 +621,23 @@
       expect(described_class.payrollable).to match_array([first_unpayrolled_claim, second_unpayrolled_claim])
     end
   end
+
+  describe "#scheduled_payment_date" do
+    let(:claim) { build(:claim, :submitted) }
+
+    context "when a claim has a payroll run associated with it" do
+      let!(:payroll_run) { create(:payroll_run, scheduled_payment_date: Date.parse("2019-01-01")) }
+      let!(:payment) { create(:payment, :with_figures, net_pay: 500.00, student_loan_repayment: 60, claims: [claim], payroll_run: payroll_run) }
```

Comment:
Could the two lines above now be simplified to just the one?

```
      let!(:payment) { create(:payment, claims: [claim], scheduled_payment_date: Date.parse("2019-01-01")) }
```

Also when a value is important to the meat of the test, i.e. in this case the date, I sometimes find it helpful to see it pulled out to a named variable, perhaps with a let:

```
  let(:scheduled_payment_date) { Date.parse("2019-01-01") }
```